integration-tests/test_connect.py

```diff
@@ -18,6 +18,7 @@
 )
 
 logger = logging.getLogger(__name__)
+pytestmark = pytest.mark.usefixtures("install_katello_rpm")
 
 
 @pytest.mark.parametrize("auth", ["basic", "activation-key"])
```

Comment:
out of curiosity: Can we run basic auth for satellite? If so then ok else we need to skip basic auth for satellite

Reply:
Basic authentication is possible with any candlepin server. Thus, it is also possible, when Satellite is used.

Reply:
Thanks George!

Reply:
Hi @jirihnidek I recall that basic auth is deprecated with insights-client. So what about the basic auth for rhc?